src/api/signals/project_edits.py

```diff
@@ -0,0 +1,48 @@
+
+from django import dispatch
+from django.db.models.signals import post_save, pre_delete
+from django.contrib.contenttypes.models import ContentType
+
+from ..models import Project, EditedProject
+from ..utils.related import get_related_project
+
+IGNORED_MODELS = [
+    "editedproject",
+    "summarysampleeventmodel",
+    "summarysitemodel",
+]
+
+
+def get_all_project_ids():
+    qs = Project.objects.filter(status__in=[Project.OPEN, Project.LOCKED])
+    return [p.pk for p in qs]
+
+
+def update_project_changes(instance, *args, **kwargs):
+    project = get_related_project(instance)
```

Comment:
Scenario: user syncs 100 SUs, each with 100 observations, after long period offline. db hits would be something like this?:
- 4 hits per observation to find project id, 3 per SU, 2 per SE, 1 per site
- 1 hit per each of above to get `EditedProjects`
- 1 hit to insert the project pk
Right? So something like (100 * 100 * 4) + (100 * 3) + (40 * 2) + (40 * 1) + (100 * 100) + 100 + 40 + 40 + 1 = 50601?

I know we decided we couldn't use `Revision` to track what's changed, but what about using the `push` endpoint to extract project ids from the request body instead of doing db calls? Hmm I guess looking there, we'd still have to do the db calls to look up... Well, dang.